src/init.cpp

```diff
@@ -588,6 +588,7 @@ std::string HelpMessage(HelpMessageMode mode)
     }
 
     strUsage += HelpMessageOpt("-maxtipage=<n>", "regtest only - Set the maximum tip age used for the relay of chain blocks)");
+    strUsage += HelpMessageOpt("-skipscopforkcheck", strprintf("regtest only - do not check deprecation of sc_create / sc_send method based on fork point (default: %d))", false));
```

Comment:
It looks like this parameter is not that useful after all. Even if we force the creation of transactions that are not allowed from the RPC, the mempool rejects them immediately.
I would suggest removing it altogether, unless I am missing something else?

Reply:
That is exactly the reason for the '`regtest`' flag: verifying via py script that an offending transaction is rejected from the mempool. We cannot rely on transactions being sent solely via the zend RPC command.

Reply:
Ok, we can remove the option, after all the consensus is already tested when a reorg is attempt, in that case the offending tx is removed from mempool.

Reply:
Done. Added also the case of sending signed raw tx after the fork point.

Reply:
I like it, thanks!